### PR TITLE
Some rendering stuff

### PR DESCRIPTION
**Pull request by OroArmor**
Also does some font stuff

### CODE REVIEW

File: mappings/net/minecraft/client/font/MultilineText.mapping [hunk 1]
```diff
@@ -4,14 +4,37 @@ CLASS net/minecraft/unmapped/C_ogfbkizf net/minecraft/client/font/MultilineText
 	METHOD m_geozwbme create (Lnet/minecraft/unmapped/C_mavozmpp;[Lnet/minecraft/unmapped/C_rdaqiwdt;)Lnet/minecraft/unmapped/C_ogfbkizf;
 		ARG 0 renderer
 		ARG 1 texts
+	METHOD m_gghylmax creat (Lnet/minecraft/unmapped/C_mavozmpp;I[Lnet/minecraft/unmapped/C_rdaqiwdt;)Lnet/minecraft/unmapped/C_ogfbkizf;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/font/MultilineText.mapping]:
```suggestion
	METHOD m_gghylmax create (Lnet/minecraft/unmapped/C_mavozmpp;I[Lnet/minecraft/unmapped/C_rdaqiwdt;)Lnet/minecraft/unmapped/C_ogfbkizf;
```
Another spell-check anomaly?

File: mappings/net/minecraft/client/render/GameRenderer.mapping [hunk 2]
```diff
@@ -308,7 +323,14 @@ CLASS net/minecraft/unmapped/C_sktkdoun net/minecraft/client/render/GameRenderer
...
 		ARG 0 shader
 	METHOD m_vwvkdyvv getPositionColorLightmapShader ()Lnet/minecraft/unmapped/C_alllhitb;
+	METHOD m_vzamypqw getPotentialHit (Lnet/minecraft/unmapped/C_astfners;DDF)Lnet/minecraft/unmapped/C_eetoegzn;
+		ARG 1 entity
+		ARG 2 blockRange
+		ARG 4 entityRange
+		ARG 6 tickDelta
 	METHOD m_vzhzwpiw getRenderTypeEntityAlphaShader ()Lnet/minecraft/unmapped/C_alllhitb;
+	METHOD m_vziixqvb (Lnet/minecraft/unmapped/C_astfners;)Z
+		ARG 0 e
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/GameRenderer.mapping]:
```suggestion
		ARG 0 entity2
```
would be more in line with other lambda params I've seen in QMap, but we should probably clarify what to do for these in a convention.

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/client/render/debug/DebugRenderer.mapping [hunk 3]
```diff
@@ -16,36 +16,16 @@ CLASS net/minecraft/unmapped/C_hlltolhp net/minecraft/client/render/debug/DebugR
...
-		ARG 2 minX
-		ARG 4 minY
-		ARG 6 minZ
-		ARG 8 maxX
-		ARG 10 maxY
-		ARG 12 maxZ
-		ARG 14 red
-		ARG 15 green
-		ARG 16 blue
-		ARG 17 alpha
```
Comment (eristhea) [inline mappings/net/minecraft/client/render/debug/DebugRenderer.mapping:35]: i love param automapping

File: mappings/net/minecraft/client/render/entity/ItemEntityRenderer.mapping [hunk 4]
```diff
@@ -5,9 +5,12 @@ CLASS net/minecraft/unmapped/C_eutooupz net/minecraft/client/render/entity/ItemE
...
+	METHOD m_jfzxzfsn renderItem (Lnet/minecraft/unmapped/C_xjqetogb;Lnet/minecraft/unmapped/C_cnszsxvd;Lnet/minecraft/unmapped/C_igrgeffe;ILnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_nnhmhkbv;ZLnet/minecraft/unmapped/C_rlomrsco;)V
 		ARG 0 renderer
-	METHOD m_xhxgqzji (Lnet/minecraft/unmapped/C_xjqetogb;Lnet/minecraft/unmapped/C_cnszsxvd;Lnet/minecraft/unmapped/C_igrgeffe;ILnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_rlomrsco;Lnet/minecraft/unmapped/C_cdctfzbn;)V
+		ARG 3 light
+		ARG 6 draw3d
+	METHOD m_rmioqppv getRandomSeed (Lnet/minecraft/unmapped/C_sddaxwyk;)I
+	METHOD m_xhxgqzji renderItem (Lnet/minecraft/unmapped/C_xjqetogb;Lnet/minecraft/unmapped/C_cnszsxvd;Lnet/minecraft/unmapped/C_igrgeffe;ILnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_rlomrsco;Lnet/minecraft/unmapped/C_cdctfzbn;)V
 		ARG 0 renderer
-	METHOD m_zbaeotrg (I)I
+	METHOD m_zbaeotrg itemsToDraw (I)I
```
Comment (eristhea) [inline mappings/net/minecraft/client/render/entity/ItemEntityRenderer.mapping]:
```suggestion
	METHOD m_zbaeotrg getItemsToDraw (I)I
```

**Review by eristhea — APPROVED** [review]
looks fantastic!

File: mappings/net/minecraft/client/render/entity/ItemEntityRenderer.mapping (hunk 4 above)
Comment (OroArmor) [reply mappings/net/minecraft/client/render/entity/ItemEntityRenderer.mapping]:
```suggestion
	METHOD m_zbaeotrg getTiledIconsToDraw (I)I
```

**Review by IotaBread — APPROVED** [review]

**Pull request merged**